### DISCUSSION TITLE
option to disable inconsistent netherite knockback

### DISCUSSION THREAD

**Discussion by MavG8ER222** [PurpurMC/Purpur#620] [__Dismissed__] [answered]
I host a server, and we do a little pvp. but the most annoying thing is that the inconsistent netherite knockback makes it so even if your good at comboing someone they will still get to hit you and your skill doesnt matter. I already tried a plugin for this but when you give someone slow falling they go up way to high. normally the player is supposed to go back down after the second hit but with the plugin they just keep going up so if you could add a option to disable inconsistent netherite knockback that would be great.

**Comment by BillyGalbreath** [comment, marked as answer]
We try to avoid getting into the combat system as much as possible. With the way you describe the issue, it sounds extremely difficult to pull off in a way that will please everyone. So, rather than get involved in that controversy we simply dont touch combat on that kind of level.
